qiskit/VERSION.txt

```diff
@@ -1 +1 @@
-0.25.0rc1
+0.45.0
```

Comment:
Should this have a dev version attached to it?

Reply:
We've never really bothered - we attach the `git` commit information programmatically on import if we can find it, but other than that we just leave it like this.